quotes.py

```diff
@@ -0,0 +1,233 @@
+                AdminTitle = "Channel"  # 100%
+            elif reply.fwd_from.from_id:
+                try:
+                    UserID = reply.fwd_from.from_id
+                    resolved = (await self.client(telethon.tl.functions.users.GetFullUserRequest(UserID))).user
+
+                    if not await self.client.get_profile_photos(UserID):
+                        ProfilePhotoURL = ""
+
+                    Username = telethon.utils.get_display_name(resolved)
+                    AdminTitle = ""
+                except ValueError:
+                    ProfilePhotoURL = ""
+                    Username = "A user."
+                    AdminTitle = "User"
+
+        if ProfilePhotoURL != "" or Channel:
+            with BytesIO() as pfp:
+                if Channel:
+                    await self.client.download_profile_photo(Channel, pfp)
+                else:
+                    await self.client.download_profile_photo(UserID, pfp)
+                pfp.seek(0)
+                ProfilePhotoURL = "data:image/png;base64, " + base64.b64encode(pfp.read()).decode()
+                pfp.close()
+
+        if UserID is not None:
+            UsernameColor = self.config["USERNAME_COLORS"][UserID % 7]
+        else:
+            UsernameColor = self.config["DEFAULT_USERNAME_COLOR"]
+
+        request = json.dumps({
+            "ProfilePhotoURL": ProfilePhotoURL,
+            "UsernameColor": UsernameColor,
+            "Username": Username,
+            "AdminTitle": AdminTitle,
+            "Text": reply.message,
+            "Markdown": get_markdown(reply),
+            "Template": args[0],
+            "APIKey": self.config["API_TOKEN"]
+        })
+
+        resp = await utils.run_sync(requests.post, self.config["API_URL"] + "/api/v2/quote", data=request)
+        resp = await utils.run_sync(resp.json)
+
+        if resp["status"] == 500:  # API RETURNS 200 STATUS CODE ALL TIME
+            return await utils.answer(message, self.strings["SERVER_ERROR"])
+        elif resp["status"] == 401:
+            if resp["message"] == "ERROR_TOKEN_INVALID":
+                return await utils.answer(message, self.strings["INVALID_TOKEN"])
+        elif resp["status"] == 403:
+            if resp["message"] == "ERROR_UNAUTHORIZED":
+                return await utils.answer(message, self.strings["UNAUTHORIZED"])
+        elif resp["status"] == 404:
+            if resp["message"] == "ERROR_TEMPLATE_NOT_FOUND":
+                newreq = await utils.run_sync(requests.post, self.config["API"] + "/api/v1/getalltemplates", data={
+                    "token": self.config["API_TOKEN"]
+                })
+                newreq = await utils.run_sync(newreq.json)
+
+                if newreq["status"] == "NOT_ENOUGH_PERMISSIONS":
+                    return await utils.answer(message, self.strings["NOT_ENOUGH_PERMISSIONS"])
+                else:
+                    templates = self.strings["DELIMITER"].join(newreq["message"])
```

Comment:
not like that.
`templates = "".join(self.strings["template_format"].format(template) for template in newreq["message"])`